core/src/main/java/org/testcontainers/dockerclient/DockerMachineClientProviderStrategy.java

```diff
@@ -16,6 +16,9 @@
  */
 @Slf4j
 public class DockerMachineClientProviderStrategy extends DockerClientProviderStrategy {
+
+    public static final int PRIORITY = EnvironmentAndSystemPropertyClientProviderStrategy.PRIORITY - 10;
```

Comment:
Ooh, but can we actually do that? For example, I would like Docker-for-Mac/Win to always come before Docker Machine. That way users at least have a chance to control which gets used from the outside (they can quit DfM and docker-machine gets used as a fallback). Similarly for setting env vars to force use of a particular docker daemon.

This might need some care 🤔...